CONTRIBUTING.rst

```diff
@@ -148,15 +171,7 @@ modules in the local ``SCons`` subdirectory tree and then run
 
     $ python scripts/scons.py [arguments]
 
-If you want to be able to just execute your modified version of SCons from the
```

Comment:
This is still a viable option..

Reply:
Sure, but it didn't seem worth a special mention to me. No problem if it's put back, either. And of course it's an instruction that doesn't apply to WIndows.

Reply:
Definitely mention after the virtualenv bit, and maybe something like "lastly if for some reason virtualenv isn't viable this will also work"
for non windows..

Reply:
I can do that. It's in git with the executable bits on already, should I make that line an add-on ("Note: you may need to chmod +x ...")